packages/quantic/.prettierrc.js

```diff
@@ -6,8 +6,8 @@ module.exports = {
     overrides: [
       {
         files: '**/lwc/**/*.html',
-        options: {parser: 'lwc'}
-      }
-    ]
-  }
+        options: {parser: 'lwc', printWidth: 140},
```

Comment:
@lbergeron Are you guys using prettier locally with this file? or was this generated by default from the LWC project?

Reply:
This is printWidth is added by me. Reverted the file. The `.prettierignore` file under root folder disables formatting for html files and css files under dir lwc